//#define CATCH_CONFIG_MAIN
//#include "catch_amalgamated.hpp"
//
//#include <string>  // Required for std::string
//#include <vector>  // Required for std::vector
//#include <iostream>
//
//TEST_CASE("Intentional Failure Test") {
//    REQUIRE(1 + 1 == 3); // This will FAIL
//}
//
//TEST_CASE("Force output test") {
//    std::cout << "This test is running!\n";
//    REQUIRE(1 + 1 == 3); // Intentional fail
//}
//
